docker-cmd-mac.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+# Example of docker usage for RLLIB + SUMO Utlis
+#
+# Author: Lara CODECA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+
+set -e
+set -u
+
+IMAGE_NAME="tf-gpu-sumo-$(date +%Y-%m-%d)"
+IMAGE_FOLDER="docker-image-mac"
+GPU=true
+GPU_OPT="--gpus all"
+OPTIRUN=false
```

Comment:
optirun does not exist for Mac OSX 
- https://wiki.archlinux.org/index.php/Bumblebee

Reply:
Updated in the commit 97d2053f9f26c49be70c077d79589d8548a32f9a